backend/src/main/resources/db/migration/common/V1_0_18__Add_Contact_Person_Profile_Photo.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE contact_person_profile
+    ADD COLUMN photo_url varchar (1030);
```

Comment:
Where does this number come from? It is just a curiosity

Reply:
Don't remember. I'll ask.

Reply:
I think It was put it that way because we don't know the url length. But maybe it's better only varchar removing the 1030?